src/CoreBundle/private/modules/CMSTemplateEngine/CMSTemplateEngine.class.php

```diff
@@ -262,33 +262,25 @@ public function &Execute()
     }
 
     /**
+     * @deprecated since 6.3.0 - revision management is no longer supported
+     *
      * loads revision management relevant data if active.
      */
     protected function LoadRevisionData()
     {
-        $this->data['bRevisionManagementActive'] = false;
```

Comment:
Maybe we should keep this one line, in case someone still relies on it.

Reply:
added again